analyses/molecular-subtyping-embryonal/01-samples-to-subset.Rmd

```diff
@@ -17,8 +17,14 @@ To summarize - we will identify biospecimen IDs that should be included for down
 1. An RNA-seq biospecimen includes a _TTYH1_ fusion (5' partner).
    We can consume the files from the [`fusion-summary` module](../fusion-summary/) for this step.
    (See also: [this comment on `AlexsLemonade/OpenPBTA-analysis#401`](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/401#issuecomment-573669727).)
-2. Any sample with `"Embryonal tumor"` in the `broad_histology` column of `pbta-histologies.tsv` that is not labeled `"Medulloblastoma"` or `"Atypical Teratoid Rhabdoid Tumor (ATRT)"` in `pathology_diagnosis` per [this comment on `AlexsLemonade/OpenPBTA-analysis#251`](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/251#issuecomment-568220913).
-   We'll look at the `integrated_diagnosis` column as well.
+   
+2. An RNA-seq biospecimen sample includes a _MN1_ fusion (5' partner) [per this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/785#issuecomment-695015488).
```

Comment:
This documentation should reflect that we are excluding the MN1--PATZ1 fusion and why.